src/main/kotlin/org/radarbase/push/integration/garmin/service/BackfillService.kt

```diff
@@ -43,7 +44,10 @@ class BackfillService(
         redisHolder,
         config.pushIntegration.garmin.backfill.redis.lockPrefix
     )
-    private val httpClient = OkHttpClient()
+    private val httpClient = OkHttpClient.Builder()
```

Comment:
Wondering whether the context OkHttpClient could also work?

Reply:
Yes would work looking at the timeouts, i will change it for now. Later if more customisations are needed, we might have to define them here again.